mw-webapp/index.html

```diff
@@ -11,6 +11,11 @@
     />
     <link rel="apple-touch-icon" href="public/logo192x192.png" />
     <link rel="manifest" href="/manifest.json" />
+
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
```

Comment:
Do we use Monseratt font in the project?  Just thought that we will use only Roboto, am I wrong?

Reply:
removed Monserrat